packages/core/src/compatibility/browser.ts

```diff
@@ -5,10 +5,10 @@
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
 
-const userAgent = navigator.userAgent;
+const userAgent = typeof navigator !== "undefined" ? navigator.userAgent : null;
```

Comment:
default to `""` and you won't have to check its existence below. `"".match(/Edge/) --> false`